js/widgets/help.js

```diff
@@ -21,6 +21,8 @@ function HelpWidget() {
         this.isOpen = true;
 
         let widgetWindow = window.widgetWindows.windowFor(this, "help", "help");
+        widgetWindow.getWidgetBody().style.overflowY = "auto";
+        widgetWindow.getWidgetBody().style.maxHeight = "500px";
```

Comment:
I think a dynamic height, e.g. `${canvas.getBoundingClientRect().height * .75}px`; would be better.

Reply:
for overflowY to work as expected, we need to specify a fixed threshold, here 500px was used because, in other widgets{timbre for instance}, it was the value set as `height` or `maxHeight` here.